packages/wonder-blocks-button/components/button.stories.js

```diff
@@ -1,12 +1,80 @@
 // @flow
 import React from "react";
+import {action} from "@storybook/addon-actions";
+import {withKnobs, text, radios, object, boolean} from "@storybook/addon-knobs";
+import {jsxDecorator} from "storybook-addon-jsx";
+
 import {View} from "@khanacademy/wonder-blocks-core";
 import {Strut} from "@khanacademy/wonder-blocks-layout";
 import Color from "@khanacademy/wonder-blocks-color";
 
 import Button from "./button.js";
 
-export default {title: "Button"};
+export default {
+    title: "Button",
+    decorators: [withKnobs, jsxDecorator],
+
+    parameters: {
+        options: {
+            showAddonPanel: false,
```

Comment:
Same with this option, it could be defined globally instead of defined on every file.